core/templates/pages/exploration-editor-page/editor-tab/state-version-history/state-version-history.component.ts

```diff
@@ -78,13 +74,21 @@ export class StateVersionHistoryComponent {
     if (stateDiffData.newState) {
       modalRef.componentInstance.newState = stateDiffData.newState;
       if (stateDiffData.newState.name === null) {
+        // The state name is null before having a state
+        // (Please refer StateObjectFactory). This cannot happen here because
```

Comment:
Please --> please

refer --> refer to the implementation of  (assuming this is what you mean? I'm not sure)

Reply:
Done.